package.json

```diff
@@ -230,7 +230,7 @@
               }
             },
             {
-              "name": "Azure Premium Disk storage",
```

Comment:
Classic GP was not using premium disks but premium page blobs. Next-Gen is actually using managed disks. So to cover both,  I switched here to Azure Remote Storage